appeals/api/src/server/tests/appeals/expectation.js

```diff
@@ -15,11 +15,11 @@ export const baseExpectedLPAQuestionnaireResponse = (appeal) => ({
 		// @ts-ignore
 		whoNotified: {},
 		// @ts-ignore
-		siteNotice: {},
+		whoNotifiedSiteNotice: {},
```

Comment:
will this change affect my previous work where I use 'siteNotice' for LPAQ?

Reply:
The test was failing - this is why I changed them. I think once part 2 is done, will be aligned...

Reply:
api test failing? if so I couldn't have merged my branch.
We have 'siteNotice' etc in `openapi.json`, not `whoNotifiedSiteNotice`